database/test/test_services.py

```diff
@@ -167,6 +181,24 @@ def test_populate_local_koodistot(populate_local_koodistot, main_db_params):
         conn.close()
 
 
+def test_populate_admin_region_geometries(
+    populate_admin_region_geometries, populate_koodistot, main_db_params
+):
+    """
+    Test that maakunta geometries are populated
+    """
+    conn = psycopg2.connect(**main_db_params)
+    try:
+        with conn.cursor() as cur:
+            cur.execute(
+                "SELECT count(*) FROM codes.administrative_region WHERE geom IS NULL"
+            )
+            geom_count = cur.fetchone()[0]
+            assert geom_count == 19
```

Comment:
@msorvoja Here I'm wondering about this test, it doesn't seem to work. Here we test that geom is NULL in all the 19 rows, it should be NOT NULL I think?

Also, when running the test, my call responds with
```
{'statusCode': 200, 'body': '"0 inserted or updated. 0 deleted."'}
```
so it looks here we are testing that the code does NOT fill any geometries.

Reply:
Good catch!